### DISCUSSION TITLE
Auth session not persisted

### DISCUSSION THREAD

**Discussion by churichard** [supabase/supabase#3116] [Questions]
I'm having an issue where my auth session is sometimes not being persisted across sessions. When I come back to my app, I see that my local storage has been cleared and I need to log in again (even though I have not cleared my cookies/cache).

I'm using Next.js + Supabase with client-side auth. Any ideas on what could be causing this issue?

I noticed this happening one time after I made a deployment (I'm using Vercel), but it could be unrelated.

**Comment by dshukertjr** [comment]
Hi @churichard!

Does this happen every time you sign in, or just once in a while?

**Reply by churichard** [reply]
It happens randomly every once in a while. I made an issue with more details: https://github.com/supabase/supabase-js/issues/254
